lib/active_merchant/billing/gateway.rb

```diff
@@ -57,7 +57,7 @@ class Gateway
       include CreditCardFormatting
 
       DEBIT_CARDS = [ :switch, :solo ]
-      CURRENCIES_WITHOUT_FRACTIONS = %w(BIF BYR CLP CVE DJF GNF HUF ISK JPY KMF KRW PYG RWF UGX VND VUV XAF XOF XPF)
+      CURRENCIES_WITHOUT_FRACTIONS = %w(BIF BYR CLP CVE DJF GNF HUF IDR ISK JPY KMF KRW PYG RWF UGX VND VUV XAF XOF XPF)
```

Comment:
Yeah I'm on the fence about it.  I'm a little cautious about updating the Gateway list since lots of gateways are calling this:

https://github.com/duff/active_merchant/blob/master/lib/active_merchant/billing/gateway.rb#L241

which is using that list.

And Wikipedia's description of the `sen` is really confusing!  https://en.wikipedia.org/wiki/Indonesian_rupiah

I'm thinking we _know_ from Worldpay's docs how they handle the `IDR` so it may be safer to only mess with Worldpay.